ext/lib/crypto/mbedtls/configs/config-mini-tls1_2.h

```diff
@@ -74,7 +74,7 @@
 #define MBEDTLS_SSL_ALL_ALERT_MESSAGES
 #endif
 
-#define MBEDTLS_SSL_MAX_CONTENT_LEN  1500
+#define MBEDTLS_SSL_MAX_CONTENT_LEN  2500
```

Comment:
Please put this into separate patch. Even better would be to add if defined(MQTT...) here if this increase is required by MQTT only.